client/fieldregistry.go

```diff
@@ -62,6 +62,7 @@ type fieldRegistry struct {
 	PasswordLastSet    *Field `ldap:"passwordLastSet"`
 	RACFID             *Field `ldap:"racfid"`
 	RACFPassword       *Field `ldap:"racfPassword"`
+	RACFPassphrase     *Field `ldap:"racfPassphrase"`
```

Comment:
Payman pointed out that the [RACF documents](https://www.ibm.com/docs/en/zos/2.4.0?topic=sob-changing-user-password-password-phrase-in-racf-using-sdbm) uses `racfPassPhrase` (camel case). We should match that convention where it makes sense.